src/recipe/DTOs/update-recipe.dto.ts

```diff
@@ -7,33 +7,41 @@ import {
 import { CreateIngredientDTO } from './create-ingredient.dto';
 import { CreatePreparingDTO } from './create-preparation.dto';
 import { Type } from 'class-transformer';
-import { ApiProperty } from '@nestjs/swagger';
+import { ApiPropertyOptional } from '@nestjs/swagger';
 
 export class UpdateRecipeDTO {
   @IsString()
   @IsOptional()
-  @ApiProperty({ example: 'Dumplings', type: 'string' })
+  @ApiPropertyOptional({ example: 'Dumplings', type: 'string' })
   name?: string;
 
   @IsString()
   @IsOptional()
-  @ApiProperty({ example: 'Easy dumplings recipe', type: 'string' })
+  @ApiPropertyOptional({ example: 'Easy dumplings recipe', type: 'string' })
   description?: string;
 
   @IsNumber()
   @IsOptional()
-  @ApiProperty({ example: 'imageKey', type: 'string' })
+  @ApiPropertyOptional({ example: 'imageKey', type: 'string' })
   fileId?: number;
 
   @ValidateNested()
   @IsOptional()
   @Type(() => CreatePreparingDTO)
-  @ApiProperty({ type: [CreatePreparingDTO], isArray: true })
+  @ApiPropertyOptional({
+    type: [CreatePreparingDTO],
+    isArray: true,
+    example: [{ name: 'add flour', step: 1 }],
+  })
   preparing?: CreatePreparingDTO[];
 
   @ValidateNested()
   @IsOptional()
   @Type(() => CreateIngredientDTO)
-  @ApiProperty({ type: [CreateIngredientDTO], isArray: true })
+  @ApiPropertyOptional({
+    type: [CreateIngredientDTO],
```

Comment:
I don't think you have to specify `type: [CreateIngredientDTO]` if isArray is set to true